docs/changelog.rst

```diff
@@ -10,18 +10,24 @@ Changes will be added here periodically from the "Suggested changelog entry"
 block in pull request descriptions.
 
 
-
 IN DEVELOPMENT
 --------------
 
 Changes will be summarized here periodically.
 
-Version 2.10.1 (Oct 2?, 2022)
+Version 2.10.1 (Oct 24, 2022)
 -----------------------------
 
+There is some concern about ABI compatibility between 2.10.x and previous
```

Comment:
Suggested revised paragraph:
________
There is significant concern about ABI compatibility between 2.10.x (including this release) and previous versions, possibly related to cross-module exception handling. This is still developing and tracked under #4105. Please review if this is important to you before upgrading, and report an issue or comment on an issue if there's a problem.
________

I think it is best not to say anything about the `internals` version bump until we have more clarity. I'm currently holding a pretty strong belief that the `internals` version bump won't help (contrary to what I was assuming a couple days ago).

Reply:
Used that, though replaced "significant" with "some".